src/olympia/reviewers/serializers.py

```diff
@@ -586,3 +580,16 @@ def validate(self, data):
                     'You can\'t submit a line number without associating '
                     'it to a filename.')})
         return data
+
+
+class DraftCommentSerializerReadOnly(DraftCommentSerializer):
```

Comment:
I think if the serializer is named `...ReadOnly` you should make all the fields be readonly.

Reply:
That makes sense. I called it `...ReadOnly` just because it's used for the `list` and `retrieve` cases, but yeah, if I'm going to use that name I should make sure it actually is readonly. Perhaps a better overall solution, which I asked about in another comment, will eliminate the need for this entirely.